.rubocop.yml

```diff
@@ -1,29 +1,169 @@
+inherit_from:
+  - .rubocop_todo.yml
+
 AllCops:
+  TargetRubyVersion: 2.1
+  TargetRailsVersion: 3.2
   Include:
     - '**/Rakefile'
     - '**/config.ru'
   Exclude:
     - 'db/**/*'
     - 'config/**/*'
     - 'script/**/*'
-    - 'spec/**/*'
     - !ruby/regexp /old_and_unused\.rb$/
 
-Documentation:
+# OFN SETTINGS
+# Cop settings that have been agreed upon by the OFN community
+
+Rails:
+  Enabled: true
+
+Style/Documentation:
+  Enabled: false
+
+Style/StringLiterals:
+  Enabled: false
+
+# TEMPORARY/CONTESTED SETTINGS
+# These are still to be decided upon, but recommended for inclusion by
+# oeoeaio after scrutinising offenses the codebase
+
+# Don't think this is a big issue, mostly picking up RPSEC scope definitions
+# with lamdas and RSpec '.to change{}' blocks
+Lint/AmbiguousBlockAssociation:
+  Enabled: false
+
+# No alternative to this one until we upgrade to Rails 4 and can use #find_by
+Rails/DynamicFindBy:
+  Enabled: false
+
+# This should be the programmer's discretion, perhaps we should review all of
+# the uses of it an make specific exceptions though.
+Rails/SkipsModelValidations:
+  Enabled: false
+
+# This should be enfored, but doing so would require changing the entire
+# file, so should probably do when outstanding PRS are at a minimum
+Style/ClassAndModuleChildren:
```

Comment:
> So if we are using Rubocop linters, and we remove this line, devs will start getting errors about things that they didn't cause.

That's why we need to add them in the rubocop_todo.yml. When running rubocop this will skip the currenr violations but it will raise the new ones not present in the todo. Code Climate will do exactly the same as it runs de rubocop as well.

The point I believe is to agree on a style guide (rubocop.yml) and "hide" all existing violations (rubocop_todo.yml) so that we can apply the guide on any new code without having to clean them up upfront.